lib/rspec/rails/matchers/routing_matchers.rb

```diff
@@ -18,8 +18,8 @@ def initialize(scope, *expected)
             end
           end
 
-          def matches?(verb_to_path_map)
-            @actual = @verb_to_path_map = verb_to_path_map
+          def matches?(actual)
+            @actual = verb_to_path_map
```

Comment:
You can't rename actual here and reference `verb_to_path_map`, either switch both to actual or leave the argument alone :)